src/components/Modal/BaseModal.tsx

```diff
@@ -249,25 +269,27 @@ function BaseModal(
                     deviceHeight={windowHeight}
                     deviceWidth={windowWidth}
                     animationIn={animationIn ?? modalStyleAnimationIn}
+                    animationInDelay={animationInDelay}
                     animationOut={animationOut ?? modalStyleAnimationOut}
                     // eslint-disable-next-line react-compiler/react-compiler
                     useNativeDriver={useNativeDriverProp && useNativeDriver}
                     // eslint-disable-next-line react-compiler/react-compiler
                     useNativeDriverForBackdrop={useNativeDriverForBackdrop && useNativeDriver}
                     hideModalContentWhileAnimating={hideModalContentWhileAnimating}
-                    animationInTiming={animationInTiming}
-                    animationOutTiming={animationOutTiming}
+                    animationInTiming={animationInTiming ?? 300}
```

Comment:
What do you think about moving the value 300 to a variable above, like this: 
`const defaultAnimationTime = 300`?
Or assigning `animationOutTiming` & `animationInTiming` default values, just like it is done with other props.